tests/tracer/test_propagation.py

```diff
@@ -170,18 +170,47 @@ def test_extract(tracer):
             assert child_span.context.dd_origin == "synthetics"
             assert child_span.context._meta == {
                 "_dd.origin": "synthetics",
-                "_dd.p.dm": "value",
+                "_dd.p.test": "value",
             }
 
 
+@pytest.mark.parametrize(
+    "x_datadog_tags, expected_trace_tags",
+    [
+        ("_dd.p.dm=193ed935a8-0", {"_dd.p.dm": "193ed935a8-0"}),
+        ("_dd.p.dm=-0", {"_dd.p.dm": "-0"}),
+        ("_dd.p.dm=193ED935A8-0", {"_dd.propagation_error": "decoding_error"}),
```

Comment:
*must* be lowercase hex?

a comment on why each of these are failing might be good.

Reply:
The format specification is for lower case.